# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../anjuta/no-pkgconfig-list-all.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We can not run --list-all, we will pull in a lot of dependencies, and the list would
not match the installed one anyway ...

  - Rene Rebe <rene@exactcode.de>

--- anjuta-1.2.3/global-tags/create_global_tags.sh	2003-12-08 07:55:22.000000000 +0100
+++ anjuta-1.2.3/global-tags/create_global_tags.sh	2005-05-20 10:59:04.000000000 +0200
@@ -41,7 +41,7 @@
 PKG_CONFIG=`which pkg-config 2>/dev/null`
 if ( [ ! -z $PKG_CONFIG ] && [ -x $PKG_CONFIG ] ) ; then
   echo "Found pkg-config ... $PKG_CONFIG"
-  PKG_CONFIG_MODULES=`$PKG_CONFIG --list-all 2>/dev/null | awk '{printf("%s ",$1);}'`
+  PKG_CONFIG_MODULES=gtk+-2.0
   PKG_CONFIG_CFLAGS=`$PKG_CONFIG --cflags $PKG_CONFIG_MODULES 2>/dev/null`
   for cflag in $PKG_CONFIG_CFLAGS
   do
